### PR TITLE
some easy doc fixes

### PR DESCRIPTION
**Pull request by craigbox**
Update readme to use docs on kubescape.io; change some governance wording from heblish.

Signed-off-by: Craig Box <craig.box@gmail.com>

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -2,11 +2,11 @@
...
-| [Rotem Refael](https://www.linkedin.com/in/rotem-refael) | [@rotemamsa](https://github.com/rotemamsa) | [ARMO](https://www.armosec.io/) | 2021-10-11 |
-| [David Wertenteil](https://www.linkedin.com/in/david-wertenteil-0ba277b9) | [@dwertent](https://github.com/dwertent) | [ARMO](https://www.armosec.io/) | 2021-09-01 |
+| Name | GitHub | Organization
+| --- | --- | --- 
+| [Matthias Bertschy](https://www.linkedin.com/in/matthias-bertschy-b427b815/) | [@matthyx](https://github.com/matthyx) | [ARMO](https://www.armosec.io/)
+| [Craig Box](https://www.linkedin.com/in/crbnz/) | [@craigbox](https://github.com/craigbox) | [Solo.io](https://www.solo.io/)
+| [Ben Hirschberg](https://www.linkedin.com/in/benyamin-ben-hirschberg-66141890) | [@slashben](https://github.com/slashben) | [ARMO](https://www.armosec.io/)
+| [Rotem Refael](https://www.linkedin.com/in/rotem-refael) | [@rotemamsa](https://github.com/rotemamsa) | [ARMO](https://www.armosec.io/)
+| [David Wertenteil](https://www.linkedin.com/in/david-wertenteil-0ba277b9) | [@dwertent](https://github.com/dwertent) | [Kaleido](https://www.armosec.io/)
```
Comment (matthyx) [inline MAINTAINERS.md:11]: change URL to kaleido.io

**Comment by matthyx** [comment]
@craigbox can you remove the `git2go` submodule from your PR, as we no longer need it

**Comment by matthyx** [comment]
@craigbox please update this PR

**Comment by craigbox** [comment]
done

**Pull request merged**